.github/container/jax_nsys/install.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Usage: ./install.sh or PYTHON=python3.X ./install.sh
+#
+# If it doesn't already exist, this creates a virtual environment named
+# `nsys_jax_env` in the current directory and installs Jupyter Lab and the
+# dependencies of the Analysis.ipynb notebook that is shipped alongside this
+# script inside the output archives of the `nsys-jax` wrapper.
+#
+# The expectation is that those archives will be copied and extracted on a
+# laptop or workstation, and this installation script will be run there, while
+# the `nsys-jax` wrapper is executed on a remote GPU cluster.
+PYTHON=${PYTHON:-python}
+if ! command -v "${PYTHON}" > /dev/null; then
+  echo "Python executable ${PYTHON} not found; try setting the PYTHON environment variable"
+  exit 1
+fi
+VIRTUALENV="${PWD}/nsys_jax_venv"
+if [[ ! -d "${VIRTUALENV}" ]]; then
+  ${PYTHON} -m venv nsys_jax_venv
+  . ./nsys_jax_venv/bin/activate
+  ${PYTHON} -m pip install -U pip
+  ./nsys-jax-ensure-protobuf
+  ${PYTHON} -m pip install jupyterlab numpy pandas protobuf pyarrow
+  curl -o "${VIRTUALENV}/bin/flamegraph.pl" https://raw.githubusercontent.com/brendangregg/FlameGraph/master/flamegraph.pl
```

Comment:
Do we want to ship the flamegraph binary in the system path instead of downloading it every time?

Reply:
This `install.sh` script is embedded in the output `.zip` archive and is intended to be run on the end-user's system, not inside the containers we build.